tests/tests_app/runners/test_cloud.py

```diff
@@ -33,6 +42,25 @@ def run(self):
         print("my run")
 
 
+class WorkWithSingleDrive(LightningWork):
+    def __init__(self):
+        super().__init__()
+        self.drive = None
+
+    def run(self):
+        print("running")
```

Comment:
the test doesn't require this in stdout, right, so it should probably be removed? also below?

Reply:
good call! removed!